### ISSUE TITLE
Check that mining pool software can connect to Zebra in CI, but don't mine any blocks

### ISSUE THREAD

**Issue by mpguerra** [ZcashFoundation/zebra#5304]
## Motivation

We want to test our RPCs against the software that uses them, to make sure we got the names, parameters, and fields correct.

## Proposed Solution

We can run the latest version of https://github.com/s-nomp/s-nomp , connect it to `zebrad`, and check for successes or failures in the logs.

This will be like the `lightwalletd_integration` test:
https://github.com/ZcashFoundation/zebra/blob/75a679792bb787b95b4e9ce87aaefe205c48c97a/zebrad/tests/acceptance.rs#L1505

## Out of Scope

We won't be mining blocks in CI, we just want to make sure the miner can connect.

**Comment by mpguerra** [comment]
I don't think this needs to block development of the RPC calls. We can start development when we are ready to start work on this, and test against the mining pool software once we have it, first manually and then in CI once this issue is complete.

**Comment by teor2345** [comment]
We decided that this was out of scope at the engineering meeting today. We only need to do this if the manual tests in #6025 aren't enough.
